File: Developer/Default.xctoolchain/cctools/ld64/unit-tests/test-cases/re-export-symbol/foo.c

#if USE_MY
  extern int mybar();
#else
  extern int bar();
#endif

int foo(void)
{
#if USE_MY
  return mybar() + 1;
#else
  return bar() + 1;
#endif
}

#if USE_MY
  void* p = &mybar;
#else
  void* p = &bar;
#endif

File: Developer/Default.xctoolchain/cctools/ld64/unit-tests/test-cases/cfstring-utf16/foo.m
#include <CoreFoundation/CFString.h>

extern void bar();

void foo()
{
	CFStringGetLength(CFSTR("hello"));
	CFStringGetLength(CFSTR("überhund"));
}


int main() 
{
	CFStringGetLength(CFSTR("über"));
	CFStringGetLength(CFSTR("überhund"));
	bar();
	return 0; 
}


